internal/test_helpers/scenarios/apiversions_length_field_too_large/your_program.sh

```
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec python3 $(dirname "$0")/main.py "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)